js/Create_Phone_Number.js

```diff
@@ -0,0 +1,11 @@
+// https://www.codewars.com/kata/create-phone-number/train/javascript
+
+function createPhoneNumber(numbers){
+    var numbersArray = numbers;
+    console.log('start');
+    if (numbersArray.length === 10){
```

Comment:
What if this condition is not met? What will the function return in that case? What should it return?

Reply:
I thin there will be return with console.log that numbers array are not equal to 10